cdap-common/src/main/java/co/cask/cdap/internal/explore/ExploreTableNaming.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.explore;
+
+import co.cask.cdap.proto.Id;
+
+/**
+ * Specifies how to name tables for Explore.
+ */
+public final class ExploreTableNaming {
+
+  public String getTableName(Id.Stream streamId) {
+    return cleanTableName(String.format("stream_%s", streamId.getId()));
```

Comment:
Shouldn't it be `return "stream_" + cleanTableName(streadId.getId());`? Meaning only clean the id?